backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/FeatureSupported.java

```diff
@@ -457,4 +457,14 @@ public static boolean isDedicatePolicySupported(Version version) {
     public static boolean isReplicateExtendSupported(Version version) {
         return Version.v4_7.lessOrEquals(version);
     }
+
+    /**
+     * Check 'virtio' driver for 'vga' display type is supported.
+     *
+     * @param version Compatibility version to check for.
+     * @return true if 'virtio' driver is supported
+     */
+    public static boolean isVirtioVgaSupported(Version version) {
+        return Version.v4_7.lessOrEquals(version);
```

Comment:
lgtm to me, but if we wish(do we? @ahadas ) to allow the user to change it, it should be added also to the engine config.

Reply:
yeah, the whole point of adding that to feature-supported was to let users disable it without code changes, so yeah, need to add it to vdc_options and then it is worth adding it also to engine-config

Reply:
@ahadas Are you sure it makes sense to be able to switch this off globally? It will rather be more useful to be able to switch it off for particual guest OSes - that's why I wanted to add this setting to osinfo.

Reply:
yeah, it can be useful when virtio-vga works properly on some guest OSes and not on other guest OSes but if that means adding another properly to osinfo (besides the existing one for video devices) then I don't think it is worth it. after all, it wasn't requested by users - we add it because we believe it's better than 'vga', if it's not the case, we can disable it globally and "revert" that decision

Reply:
Done.